doc/source/whatsnew/v1.3.0.rst

```diff
@@ -272,6 +272,7 @@ I/O
 - Bug in :func:`json_normalize` resulting in the first element of a generator object not being included in the returned ``DataFrame`` (:issue:`35923`)
 - Bug in :func:`read_excel` forward filling :class:`MultiIndex` names with multiple header and index columns specified (:issue:`34673`)
 - :func:`pandas.read_excel` now respects :func:``pandas.set_option`` (:issue:`34252`)
+- Bug in :func:`read_json` when ``orient="split"`` does not maintan numeric string index (:issue:`28556`)
```

Comment:
Bug in :func:`read_json` when ``orient="split"`` does not maintain numeric string index (:issue:`28556`)

Reply:
Done.